enterprise/server/backends/authdb/authdb.go

```diff
@@ -339,6 +351,32 @@ func (d *AuthDB) fillDecryptedAPIKey(ak *tables.APIKey) error {
 	return nil
 }
 
+func (d *AuthDB) fillChildGroupIDs(ctx context.Context, akg *apiKeyGroup) error {
+	// If the group is not designated as a parent then don't bother doing a
+	// query.
+	if !akg.IsParent {
+		return nil
+	}
+	if akg.GetCapabilities()&int32(akpb.ApiKey_ORG_ADMIN_CAPABILITY) == 0 {
```

Comment:
do we not have a "HasCapability" util method yet? We should add one to prevent errors in bit checking

Reply:
This is a bit of a special case because we typically handle capabilities checks at a higher level where we do have a HasCapability function whereas we are still dealing with the raw capabilities here. I added one here as well but it might not get a lot of mileage.